### ISSUE TITLE
Feature suggestion, embeded content from links.

### ISSUE THREAD

**Issue by coldwinds** [zim-desktop-wiki/zim-desktop-wiki#1373]
Hello,

Since we can put links of some notes into one note, could we choose to also embed the content of the links into the note (or quick content preview of a link)? like iframe of html, or quote directly from some links's content.

**Comment by pfalcon** [comment]
Yes, the latter. It's called "clipboard". Copy needed content from the old note, paste into the new note. Voila.

**Comment by coldwinds** [comment]
> Yes, the latter. It's called "clipboard". Copy needed content from the old note, paste into the new note. Voila.

Would it update automatically when the linked note updated?

**Comment by pfalcon** [comment]
No, but you can update it yourself if just linking to is not enough, and it can be used right away, without waiting for someone to implement it for you (potentially very long), or implementing it yourself (potentially, spending a lot of effort on that, which in the end may not pay off).

**Comment by quesada** [comment]
This would be lovely for long-form writing. If you write say a book, and have separate notes per chapter, but want to, at some point, see or export the entire book as a single note. I see the use case

**Comment by Aikatsui** [comment]
This feature has on onenote. No any FOSS tool has come close to OneNote
